src/Generator.Rewrite/AssemblyRewriter.cs

```diff
@@ -166,20 +194,34 @@ private void RemoveNativeSignatures(TypeDefinition type, IEnumerable<MethodDefin
             }
         }
 
-        private void RewriteWrapperMethods(TypeDefinition type, FieldDefinition entryPoints, IEnumerable<MethodDefinition> entrySignatures)
+        private void RewriteWrapperMethods
+        (
+            TypeDefinition type,
+            FieldDefinition entryPoints,
+            IEnumerable<MethodDefinition> entrySignatures
+        )
         {
-            var wrapperSignatures = type.Methods.Where(m => m.IsPublic && m.CustomAttributes.Any(a => a.AttributeType.Name == AttributeNames.AutoGenerated)).ToList();
+            var wrapperSignatures = type.Methods.Where(m =>
+            {
+                return m.IsPublic && m.CustomAttributes.Any(a => a.AttributeType.Name == AttributeNames.AutoGenerated);
+            }).ToList();
 
             foreach (var wrapper in wrapperSignatures)
             {
                 var autoGenerated = wrapper.GetCustomAttribute(AttributeNames.AutoGenerated, throwIfNoneFound: false);
                 if (autoGenerated != null)
                 {
-                    string signatureName = (string)autoGenerated.Fields.First(f => f.Name == "EntryPoint").Argument.Value;
+                    string signatureName = (string)autoGenerated
+                        .Fields
+                        .First(f => f.Name == "EntryPoint")
+                        .Argument
+                        .Value;
+
                     var signature = entrySignatures.FirstOrDefault(s => s.Name == signatureName);
-                    var slot = GetSlot(signature);
+                    int slot = GetSlot(signature);
```

Comment:
Why to change var to int?

Reply:
It's just personal preference. I just like using the type names explicitly, especially if its one of the "basic" types like numeric types, bools, or strings.

IMO this makes it easier to see the type, whereas `var slot = GetSlot(signature)` could've easily been something that returns a `Slot` object or something like that too.

Reply:
Guidelines say that we should use `var` whenever possible. Unless it's important to know that `slot` is an int in this situation, `var` should be used.